usecases/traverser/hybrid/searcher.go

```diff
@@ -107,37 +107,34 @@ func (s *Searcher) Search(ctx context.Context) (Results, error) {
 
 		if alpha < 1 {
 			res, err := s.sparseSearch()
-			if err != nil {
-				return nil, err
-			}
+			if err == nil {
 
-			found = append(found, res)
-			weights = append(weights, 1-alpha)
+				found = append(found, res)
+				weights = append(weights, 1-alpha)
+			}
```

Comment:
Is this error (and the 2 below) just being ignored? or is it propagating somewhere?

Reply:
They are being discarded

Reply:
Error propogation will require modifying quite. few functions, we'll do it for the next release